miner/worker.go

```diff
@@ -1095,13 +1109,35 @@ func (w *worker) commitTransactions(txs *types.TransactionsByPriceAndNonce, coin
 func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64) {
 	w.mu.RLock()
 	defer w.mu.RUnlock()
+
 	tstart := time.Now()
 	parent := w.chain.CurrentBlock()
 
 	if parent.Time() >= uint64(timestamp) {
 		timestamp = int64(parent.Time() + 1)
 	}
+
 	num := parent.Number()
+	// we are the megabundle worker
+	var (
+		maybeMB types.MegaBundle
```

Comment:
Why is this called maybeMB? I expected it to be a pointer.

Reply:
I thought about pointer too but holding onto the pointer is wrong cause the pointer itself will point to something else, so I want the full value copy